src/scenic/syntax/scenic.gram

```diff
@@ -1278,22 +1279,85 @@ is_bitwise_or: 'is' a=bitwise_or { (ast.Is(), a) }
 # -----------------
 
 bitwise_or:
+    | scenic_bitwise_or
     | a=bitwise_or '|' b=bitwise_xor { ast.BinOp(left=a, op=ast.BitOr(), right=b, LOCATIONS) }
     | bitwise_xor
 
+scenic_bitwise_or:
```

Comment:
These rule names are confusing to me, since there are no bitwise operators in Scenic. Why not call this `scenic_can_see`, and similarly for the other operators?

Reply:
Here, I meant "scenic operators with the same precedence as bitwise or". For example, we _can_ think of adding more operators with the same precedence (like `+` and `-`).

However, now I think about it, there won't be such cases because we usually want one operator to bound tighter than another. I will change those names to more specific ones.

Reply:
Fixed in 98f59d79489e08bab00cd62615b3fa2a1ab3ef8f